javascript/ql/src/Security/CWE-078/ShellCommandInjectionFromEnvironment.qhelp

```diff
@@ -0,0 +1,108 @@
+<!DOCTYPE qhelp PUBLIC
+"-//Semmle//qhelp//EN"
+"qhelp.dtd">
+<qhelp>
+<overview>
+	<p>
+
+		Dynamically constructing a shell command with values from the
+		local environment, such as absolute file paths, may inadvertently
+		change the meaning of the shell command. This can result in the shell
+		command misbehaving, or even allow a malicious user to execute
```

Comment:
After "misbehaving", it would be good to give a brief indication of what could go wrong. You might, for instance, add "for example if the file path contains spaces".

Reply:
I see that you have a paragraph about this below. I'd suggest combining the two.